src/constants/constants.ts

```diff
@@ -151,3 +154,44 @@ export const tenantDisplayName = 'Microsoft';
 export const firewallErrorMessage = 'To enable access, use the Windows Azure Management Portal or run sp_set_firewall_rule on the master database to create a firewall rule for this IP address or address range.';
 export const windowsResourceClientPath = 'SqlToolsResourceProviderService.exe';
 export const unixResourceClientPath = 'SqlToolsResourceProviderService';
+
+// Azure Functions
+export const azureFunctionsExtensionName = 'ms-azuretools.vscode-azurefunctions';
+export const sqlConnectionString = 'SqlConnectionString';
+export const genericCollectionImport = 'using System.Collections.Generic;';
+export const defaultSqlBindingTextLines =
+	[
+		'log.LogInformation(\"C# HTTP trigger function processed a request.\");',
+		'string name = req.Query[\"name\"];',
+		'string requestBody = await new StreamReader(req.Body).ReadToEndAsync();',
+		'dynamic data = JsonConvert.DeserializeObject(requestBody);',
+		'name = name ?? data?.name;',
+		'string responseMessage = string.IsNullOrEmpty(name) ? \"This HTTP triggered function executed successfully. Pass a name in the query string or in the request body for a personalized response.\" : $\"Hello, {name}. This HTTP triggered function executed successfully.\";'
+	];
+export const defaultBindingResult = 'return new OkObjectResult(responseMessage);';
+export const sqlBindingResult = `return new OkObjectResult(result);`;
+export const azureFunctionLocalSettingsFileName = 'local.settings.json';
+export const sqlExtensionPackageName = 'Microsoft.Azure.WebJobs.Extensions.Sql';
+export function jsonParseError(error: string, line: number, column: number): string {
+	return localize('jsonParseError', '{0} near line "{1}", column "{2}"', error, line, column);
+}
+export function failedToParse(errorMessage: string): string {
+	return localize('failedToParse', 'Failed to parse "{0}": {1}.',
+		azureFunctionLocalSettingsFileName, errorMessage);
+}
+export function settingAlreadyExists(settingName: string): string {
+	return localize('SettingAlreadyExists', 'Local app setting \'{0}\' already exists. Overwrite?', settingName);
+}
+export const yesString = localize('yesString', 'Yes');
+export const noAzureFunctionsProjectsInWorkspace = localize('noAzureFunctionsProjectsInWorkspace', 'No Azure functions projects found in the workspace');
+export const hostFileName = 'host.json';
+export const sqlBindingsHelpLink = 'https://github.com/Azure/azure-functions-sql-extension/blob/main/README.md';
+export const addPackageReferenceMessage = localize('addPackageReferenceMessage', 'To use SQL bindings, ensure your Azure Functions project has a reference to {0}', sqlExtensionPackageName);
+export const moreInformation = localize('moreInformation', 'More Information');
+export const addSqlBindingPackageError = localize('addSqlBindingPackageError', 'Error adding Sql Binding extension package to project');
+export const checkoutOutputMessage = localize('checkoutOutputMessage', 'Check output pane for more details');
```

Comment:
It's generally helpful to make `output pane` a link to actually show the output channel with messages like this since otherwise the user has no idea what channel to look in.

Reply:
removed this with other constants that are not needed anymore

Reply:
did not need this as the function that was using it was deleted